lessons/operator/10/README.md

```diff
@@ -221,21 +230,114 @@ You can configure these in the following ways:
   check: *
   ```
 
+- Silence a any check on a specific entity:
+
+  ```
+  subscription: entity:1-424242
+  check: app-health
+  ```
+
 ### EXERCISE 2: Bulk Silencing
 
 #### Scenario
 
-You are responding to an incident or managing a maintenance operation involving multiple nodes.
-You want to silence all the nodes at once instead of having to silence each one individually.
+Here’s the situation. You are responding to an incident or managing a maintenance operation that requires taking a server offline for a period of time. 
+You want to silence all the checks running on that host at once instead of having to silence each one individually.
+
 
 #### Solution
 
-To accomplish this we will configure a silence using a wider subscription, which will silence events coming from _all_ subscribed hosts, instead of just one specific host.
+To accomplish this we will configure a silence using a wildcard check name, and the entity specific subscription. This wiil silence 
+events coming from all checks running on the named entitied, instead of just one specific check.
```

Comment:
```suggestion
To accomplish this we will configure a silence using a wildcard check name, and the entity subscription. 
This will silence events coming from all checks running on the named entity, instead of just one specific check.
```